src/config.h

```diff
@@ -182,7 +186,8 @@ void RSConfig_AddToInfo(RedisModuleInfoCtx *ctx);
 #define DEFAULT_MAX_RESULTS_TO_UNSORTED_MODE 1000
 #define SEARCH_REQUEST_RESULTS_MAX 1000000
 #define NR_MAX_DEPTH_BALANCE 2
-#define MAX_DIALECT_VERSION 3
+#define MIN_DIALECT_VERSION 1 // MIN_DIALECT_VERSION is expected to change over time as dialects become deprecated.
+#define MAX_DIALECT_VERSION 3 // MAX_DIALECT_VERSION may not exceed MIN_DIALECT_VERSION + 63.
```

Comment:
If we can expose these using DEBUG command, would be nice for tests
(https://github.com/RediSearch/RediSearch/blob/9bff8bfd6f8f6448402d06477c6668643a65e72b/tests/pytests/test_dialect.py#L7)

Reply:
If this is available using `INFO MODULES` then no need to add it to `FT.DEBUG`
(tests can extract min and max dialect from info)